### PR TITLE
Revised Interface

### PR DESCRIPTION
**Pull request by Shadowtail117**
- All images have been fixed so they display properly. In addition, outdated images have been updated to be more current. 

- Most of the lists have been updated to be more clear and comprehensive, as well as to better match the information they represent. 

- Many, many grammatical, stylistic, etc. fixes.

-----

I might have missed some American/British English conflicts, so point them out if you spot them. Thank you! 

P.S. You should probably just view the file on my fork if you want to see the rich diffs; the pictures don't render otherwise.

### CODE REVIEW

**Comment by RaikireHiuduo** [comment]
Context for the old PR:  #308

File: wiki/Interface/en.md [hunk 1]
```diff
@@ -1,227 +1,293 @@
...
 -   \[6\] A random useful tip is displayed below the menu.
--   \[7\] In the lower-left is a link to the osu! website.
--   \[8\] Connection result to Bancho!
--   \[9\] In the bottom right is are the [chat](Internet_Relay_Chat "wikilink") controls (F8&F9).
--   \[10\] In the upper-right is the osu! jukebox which plays the songs in random order.
--   \[11\] Next to the userpanel, the number of beatmaps, current running time of the program and your computer clock time.
--   \[12\] In the upper-left is your user panel, click on it to display User Options.
+-   \[7\] In the lower-left is a link to the osu! website, as well as copyright information.
+-   \[8\] Connection result to Bancho! In this picture it is not shown, but the connection result looks like a chain link. 
+-   \[9\] In the bottom right are the chat controls (`F8` & `F9`).
```
Comment (oralekin) [inline wiki/Interface/en.md]: Change the places of `F9` and `F8` so they correspond with the button placings?

File: wiki/Interface/en.md [hunk 2]
```diff
@@ -1,227 +1,297 @@
...
+- Country Ranking\*
+- Global Ranking
+- Global Ranking (Selected Mods)\*
+- Friend Ranking\*
+
+Settings with an asterisk require you to be an [osu!supporter](/wiki/osu!supporter/) to access them.
+
+Click the word bubble icon to call up the **Quick Web Access** screen for the selected beatmap:
+
+-   Press `1` or click the "Beatmap Listing/Scores" button and your default internet browser will pull up the Beatmap Listing and score page of the beatmap set the selected beatmap belongs to.
```
Comment (MegaApplePi) [inline wiki/Interface/en.md]: use code for the button names
`` `Beatmap Listing/Scores` ``
do the same below

File: wiki/Interface/en.md [hunk 3]
```diff
@@ -1,227 +1,297 @@
...
+-   Press `2` or click "Beatmap Topic" and your default itnternet browser will pull up the forum thread of the beatmap set the selected beatmap belongs to.
+-   Press `3` or click "Quick Reply" and your default irnternet browser will pull up a page where you can compose a reply to the forum thread of the beatmap set the selected beatmap belongs to.
+-   Press `4` or Esc or click "Cancel" to return to the Song Selection Screen.
```
Comment (MegaApplePi) [inline wiki/Interface/en.md]: use `` `Esc` or click `Cancel` ``

File: wiki/Interface/en.md [hunk 4]
```diff
@@ -1,227 +1,297 @@
...
 ### Gameplay toolbox
 
-<img src="soloplay7.jpeg" title="fig:soloplay7.jpeg" alt="soloplay7.jpeg" width="400" /> <img src="Mode screen.png" title="fig:List of available Game modes" alt="List of available Game modes" width="100" /> This section can be called gameplay toolbox. We will cover each button usages from left to right order.
+![List of available game modes](img/Mode_screen.png)
+![](img/Soloplay7.jpeg)
+
+This section can be called the gameplay toolbox. We will cover each button's use from left to right.
 
-Pressing ESC or click the Back button to return to Main Menu
+Press ESC or click the Back button to return to Main Menu
```
Comment (MegaApplePi) [inline wiki/Interface/en.md]: `` `Esc` ``

File: wiki/Interface/en.md [hunk 5]
```diff
@@ -1,227 +1,297 @@
...
 
-Click on the "Mode" button to **open up a list of gameplay modes available on osu!**. Click on your desired gameplay mode, and osu! will switch to that gameplay mode style along with the scoreboard change. Alternatively, you can press CTRL and 1 (![](osu.gif "fig:osu.gif")[osu! Standard](Standard "wikilink")) / 2 (![](taiko.gif "fig:taiko.gif")[Taiko](Taiko "wikilink")) / 3 (![](Ctb.gif "fig:Ctb.gif")[Catch the Beat](Catch_the_Beat "wikilink")) / 4 (![](mania.gif "fig:mania.gif")[osu!mania](osu!mania "wikilink")) on your keyboard to quickly switch to your desired gameplay mode.
+Click on the `Mode` button to open up a list of gameplay modes available on osu!. Click on your desired gameplay mode, and osu! will switch to that gameplay mode style - the scoreboard will change accordingly. Alternatively, you can press CTRL and 1 (osu!standard), 2 (Taiko), 3 (osu!mania), or 4 (Catch the Beat) to change the gamemode.
```
Comment (MegaApplePi) [inline wiki/Interface/en.md]: - `` `Ctrl` ``
- `` `1` `` .. etc.
- update game mode names

File: wiki/Interface/en.md [hunk 6]
```diff
@@ -1,227 +1,297 @@
...
+Click the "Mods" button or press F1 to call up the **[Mod Selection Screen](/wiki/Game_Modifiers/)**.
 
-Click the "Random Map" button or press F2 to have the game **select a random beatmap from the currently displayed list.**
+In this screen, you can apply modifications ("mods" for short) to gameplay. Some mods lower difficulty and apply a multiplier that lowers the score you achieve. Conversely, some mods increase the difficulty, but apply a multiplier that increases the score you achieve. Finally, some mods modify gameplay in a different way. [Relax](/wiki/Game_Modifiers/#Relax) and [Auto Pilot](/wiki/Game_Modifiers/#Auto_Pilot) fall in that category.
+Place your mouse on a mod's icon to see a short description of its effect. Click on an icon to select or deselect that mod. Some mods, like Double Time, have multiple variations; click on the mod again to cycle through. The score multiplier value displays the combined effect the multipliers of the mod(s) of you have selected will have on your score. Click "Reset all mods" or press 1 to deselect all currently selected mods. Click "Close" or press 2 or Esc to return to the Song Selection Screen.
```
Comment (MegaApplePi) [inline wiki/Interface/en.md]: - `"Reset all mods"` -> `` `Reset all mods` ``
- `1` -> `` `1` ``
- `2` -> `` `2` ``
- `Esc` -> `` `Esc` ``

Comment (MegaApplePi) [inline wiki/Interface/en.md]: - `"Reset all mods"` -> `` `Reset all mods` ``
- `1` -> `` `1` ``
- `2` -> `` `2` ``
- `Esc` -> `` `Esc` ``

**Pull request merged**